engine/verification/assigner/consumer.go

```diff
@@ -75,17 +75,19 @@ func (w *Worker) Run(job module.Job) {
 // processed by the given blockID
 // the worker will translate the block ID into jobID and notify the consumer
 // that the job is done.
-func (w *Worker) FinishProcessing(blockID flow.Identifier) {
+func (w *Worker) Notify(blockID flow.Identifier) {
 	jobID := blockIDToJobID(blockID)
 	w.consumer.NotifyJobIsDone(jobID)
 }
 
-// finishProcessing is for the worker's underneath engine to report a block
-// has been processed without knowing the job queue
-// it's a callback so that the worker can convert the block id into a job
+// ProcessingNotifier is for the worker's underneath engine to report an entity
+// has been processed without knowing the job queue.
+// It is a callback so that the worker can convert the entity id into a job
 // id, and notify the consumer about a finished job.
-type finishProcessing interface {
-	FinishProcessing(blockID flow.Identifier)
+//
+// At the current version, entities used in this interface are chunks and blocks ids.
+type ProcessingNotifier interface {
```

Comment:
The purpose of name refactoring:
1. The interface being _exportable_ so that we can have its _mock_ in a separate package. 
2. Naming follows the golang convention of _single method interfaces_, e.g., 
```
type Writer interface {
    Write(p []byte) (n int, err error)
}
```